app/_src/production/deployment/multi-zone.md

```diff
@@ -69,13 +69,28 @@ A multi-zone deployment includes:
   - Send zone ingresses down to zone control plane.
   - Keep an inventory of all data plane proxies running in all zones (this is only done for observability but is not required for operations).
   - Reject connections from data plane proxies.
+  - Leader
+    - Writes to the store
```

Comment:
But you can also write stuff to a follower, for example PUT request to create a policy can go to any instance